fluid/language_model/train.py

```diff
@@ -96,39 +90,56 @@ def train(train_reader,
             staircase=True))
     sgd_optimizer.minimize(avg_cost)
 
+    # Initialize executor
     place = fluid.CUDAPlace(0) if use_cuda else fluid.CPUPlace()
     exe = fluid.Executor(place)
-
     exe.run(fluid.default_startup_program())
+
+    train_exe = fluid.ParallelExecutor(use_cuda=True, loss_name=avg_cost.name)
```

Comment:
This is a significant change. Does ParallelDo and ParallelExecutor give the same result for this model?

Reply:
no， the  result are not same as before，I will submit these later.

Reply:
https://github.com/PaddlePaddle/paddle-ce-latest-kpis/pull/116